Rmd/DESeq2_report_new.Rmd

```diff
@@ -261,50 +261,43 @@ fig <- fig %>% layout(
 fig
 ```
 
+
 # PCA plots {.tabset .tabset-fade}
 
 ```{r PCA_function, collapse=TRUE}
 
-# Useful links...
-# https://stackoverflow.com/questions/45756446/interactively-change-color-of-points-in-traces-in-r-plotly
-# https://stackoverflow.com/questions/68471610/r-plotly-dropdown-variable-selection-with-color
-# https://plotly.com/r/dropdowns/
-# https://stackoverflow.com/questions/63940025/menu-or-buttons-to-select-by-which-variable-style-the-plot-in-r-plotly
-# https://stackoverflow.com/questions/51261228/interactively-select-a-grouping-variable-in-plotly
-# https://plotly.com/r/custom-buttons/#update-button
-# https://plotly-r.com/client-side-linking.html
-# https://meghansaha.github.io/NYS_MH_Programs/
-# https://www.thetidytrekker.com/post/dull-dashboards
-
-# https://rstudio.github.io/crosstalk/using.html
 
 plot_interactive_PCA <- function(rld_data, intgroup_to_plot, all_intgroups){
   ## Perform PCA analysis
   data_pca <- plotPCA(rld_data, intgroup = all_intgroups, ntop = nrow(assay(rld_data)), returnData = TRUE)
   ## Get percent of variance explained
   percentVar <- round(100 * attr(data_pca, "percentVar"))
   ## Plot
-  highlight_pca <- highlight_key(data_pca, ~group) # is this going to fail if the data doesn't have a "group" variable? "group" too right?
+  highlight_pca <- highlight_key(data_pca, ~group) # is this going to fail if the data doesn't have a "group" variable?
```

Comment:
Why hard-code ~group?

Reply:
I thought you wrote this line

Reply:
lol that could be!

I will reconcile this when I take a closer look at intgroups, groups, batch variables, treatment variables... hopefully wrap that up this week with some kind of better solution (which also doesn't break anything)